src/screens/ContactScreen/elements/Search.js

```diff
@@ -9,7 +9,7 @@ import {
   View
 } from 'react-native';
 
-import IcClearCircle from '../../../assets/icons/ic_clear_circle';
+import IconClose from '../../../assets/icon/IconClose';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `IconClose` has been changed from `IcClearCircle`. Ensure that the new icon `IconClose` is the correct one to be used in this context and it exists in the specified path.
```diff
- import IcClearCircle from '../../../assets/icons/ic_clear_circle';
+ import IconClose from '../../../assets/icon/IconClose';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->